dist/src/test/java/io/camunda/zeebe/shared/management/ControlledActorClockEndpointTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.shared.management;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.byLessThan;
+
+import io.camunda.zeebe.shared.management.ActorClockEndpoint.Response;
+import io.camunda.zeebe.util.sched.clock.ControlledActorClock;
+import java.time.Instant;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.InstanceOfAssertFactory;
+import org.assertj.core.api.ObjectAssert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+final class ControlledActorClockEndpointTest {
+
+  private final InstanceOfAssertFactory<Response, ObjectAssert<Response>> instanceOfRecord =
```

Comment:
:+1: I like the use of the factory instead of casting the body, the error message is much nicer. It shows good forethought on your part. We should do this more often where applicable.